<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CogWheelUserListDialog</class>
 <widget class="QDialog" name="CogWheelUserListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Users</string>
  </property>
  <widget class="QPushButton" name="newButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QListWidget" name="userList">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>141</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
